.github/workflows/ios.yaml

```diff
@@ -0,0 +1,29 @@
+name: tuist build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: Build default scheme
+    runs-on: macos-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Tuist
+        run: |
+          bash <(curl -Ls https://install.tuist.io)
```

Comment:
Feels risky!

Reply:
At time of writing, the script this execs is:
```bash
#!/bin/bash

curl -LSs --output /tmp/tuistenv.zip https://storage.googleapis.com/tuist-releases/latest/tuistenv.zip

echo "Installing Tuist"
unzip -o /tmp/tuistenv.zip -d /tmp/tuistenv > /dev/null
mv /tmp/tuistenv/tuistenv /usr/local/bin/tuist
ln -sf /usr/local/bin/tuist /usr/local/bin/swift-project

chmod +x /usr/local/bin/tuist

rm -rf /tmp/tuistenv
rm /tmp/tuistenv.zip

echo "Tuist installed. Try running 'tuist'"
echo "Check out the documentation at https://tuist.io/docs"%
```

We could fetch the release directly if we wanted.

This all happens in a container on GitHub infrastructure, though, so it _should_ be low risk. 🙂

Reply:
This script lives here: https://github.com/tuist/tuist/blob/main/script/install

Of course, we do have to trust tuist.io as well. But then, if we're going to use binary they distribute, we're putting a lot of trust in them already.

Reply:
This looks good to me. It is unsafe to put that much trust on `tuist` but if we host our own binary then it would be another thing to maintain when we are starting the project. We can later on revisit to have our binary hosted somewhere.